aspnet-core/TreeGrid/Cells.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Cells | TreeGrid | ASP.NET Core | Syncfusion
-description: cells
+description: Learn here about Cells in Syncfusion Essential ASP.NET Core TreeGrid Control, its elements, and more.
 platform: aspnet-core
 control: TreeGrid
 documentation: ug
 ---
 
-# Cell
+# Cells in TreeGrid ASP.NET Core 
```

Comment:
Change the H1 like as
Cells in ASP.NET Core TreeGrid

Platform name should come first. Check and change in all places.

Reply:
Changed